src/python/pants/init/specs_calculator.py

```diff
@@ -75,56 +66,47 @@ def create(
     exclude_patterns: Optional[Iterable[str]] = None,
     tags: Optional[Iterable[str]] = None,
   ) -> Specs:
-    # Determine the literal specs.
     specs = cls.parse_specs(
       raw_specs=options.specs,
       build_root=build_root,
       exclude_patterns=exclude_patterns,
       tags=tags,
     )
 
-    # Determine `Changed` arguments directly from options to support pre-`Subsystem`
-    # initialization paths.
-    changed_options = options.for_scope('changed')
-    changed_request = ChangedRequest.from_options(changed_options)
-
-    # Determine the `--owner-of=` arguments provided from the global options
+    changed_options = ChangedOptions.from_options(options.for_scope('changed'))
```

Comment:
The main reason I inlined `changed_options = options.for_scope('changed')` was that I wanted to use the variable name `changed_options` to refer to what used to be called `ChangedRequest` and is now `ChangedOptions`. 

I couldn't come up with a good name for `options.for_scope('changed')`, and it's simple enough of an expression, that I ended up inlining.